dashboard/database.py

```diff
@@ -218,4 +218,54 @@ def get_album_sales_by_album(_conn: Connection, album_name: str):
         cur.execute(query, (album_name, ))
         data = cur.fetchall()
 
-    return data
+    return pd.DataFrame(data)
+
+
+def get_all_tag_names(_conn: Connection):
```

Comment:
Missing a return hint

```suggestion
def get_all_tag_names(_conn: Connection) -> list[str]:
```